facepaint/src/app/get-user/get-user.component.html

```diff
@@ -1,11 +1,58 @@
 <p>get-user works!</p>
 <button (click)="getUsers()"> Get Users</button>
+<style>
```

Comment:
Bad practice. If you want to style it, put styling in the get-user.component.css